client/shipping-label/components/label-purchase-modal/packages-step/style.scss

```diff
@@ -1,51 +1,78 @@
-.wcc-packages-container {
+&.dialog.card.packages-step__dialog {
+	top: -46px;
+
+	@include breakpoint( ">480px" ) {
+		top: 10%;
+		right: calc(50% - 200px);
+		bottom: 10%;
+		left: calc(50% - 200px);
+		width: 400px;
+	}
+
+	.packages-step__dialog-package-option {
+		font-weight: normal;
```

Comment:
Looks like the font-weight and margin changed here (from just `font-weight:bold` in `.wcc-move-item-dialog__package-name`) - what's the reason?